src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.CURLcode.cs

```diff
@@ -1,11 +1,9 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-
 internal static partial class Interop
 {
-    internal static partial class libcurl
+    internal static partial class LibCurl
```

Comment:
I think this should be named Http. Feels a little more at home next to Sys and Crypto. Also, libcurl vs. LibCurl in the transition is confusing